test/unit/app/managers/test_landing.py

```diff
@@ -64,7 +71,9 @@ def test_tool_landing_requests_requires_matching_client_secret(self):
         assert exception is not None
 
     def test_tool_landing_requests_get_requires_claim(self):
-        landing_request: ToolLandingRequest = self.landing_manager.create_tool_landing_request(self._tool_request)
```

Comment:
Shouldn't this be a separate test for a second path through the code?

Reply:
I think this is resolved by making the public/private distinction explicit ? I'm catching `ItemMustBeClaimed` here now.